charts/ctlog/templates/createctconfig-job.yaml

```diff
@@ -12,6 +12,16 @@ spec:
       serviceAccountName: {{ template "ctlog.serviceAccountName.createctconfig" . }}
       restartPolicy: Never
       automountServiceAccountToken: {{ .Values.createctconfig.serviceAccount.mountToken }}
+      initContainers:
+        - name: "wait-for-createtree-configmap"
+          image: "{{ template "ctlog.image" .Values.createctconfig.initContainerImage.curl }}"
+          imagePullPolicy: {{ .Values.createctconfig.initContainerImage.curl.imagePullPolicy }}
+          command: ["sh", "-c", "until curl -f -H \"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)\" --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt --max-time 10 https://kubernetes.default.svc/api/v1/namespaces/$(NAMESPACE)/configmaps/{{ template "ctlog.config" . }} | grep '\"treeID\":'; do echo waiting for Configmap {{ template "ctlog.config" . }}; sleep 5; done;"]
```

Comment:
nit:
can we use the full flag name, s/-f/--fail/
and
s/-H/--header/

for better readability